pom.xml

```diff
@@ -515,6 +515,7 @@
         <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpclient</artifactId>
         <version>${http.component.version}</version>
+        <scope>provided</scope>
```

Comment:
Because having it in `provided` scope means it can be used in `src/main/java` classes. Making it `test` scope restrict it to be used in `src/test/java` and will never get shipped.